cuml/random_forest_demo_mnmg.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
Just for consistency, should we use the suffix `_mnmg_demo` or `_demo_mnmg`? I used `_mnmg_demo` for kmeans but I don't mind changing it. Would like to be consistent, though.